verification/curator-service/api/src/model/source.ts

```diff
@@ -31,31 +31,46 @@ const fieldSchema = new mongoose.Schema({
     },
 });
 
-interface Parsing {
+interface RegexParsing {
     fields: Array<Field>;
 }
 
-const parsingSchema = new mongoose.Schema({
+const regexParsingSchema = new mongoose.Schema({
     fields: [fieldSchema],
 });
 
+interface Parser {
+    awsLambdaArn: string;
+}
+
+const parserSchema = new mongoose.Schema({
+    awsLambdaArn: {
+        type: String,
+        required: 'Enter a parser AWS Lambda ARN',
```

Comment:
do you plan on validating the format? (ex. the prefix arn:aws:us-east2)

Reply:
Yep -- had planned to add validations (and tests therearound) in a subsequent PR.